osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -743,6 +795,9 @@ protected override bool OnClick(ClickEvent e)
                 if (!Active.Value)
                     RequestScroll?.Invoke(this);
 
+                //Kill focus on SearchTextBox
+                Scheduler.Add(() => GetContainingInputManager().ChangeFocus(null));
```

Comment:
Why is focus being killed here?

Good comments don't restate the code; good comment explain why the code is doing what is doing.

Reply:
I want to make search bar always active as suggested by @peppy [here](https://github.com/ppy/osu/pull/23414#issuecomment-1550885221). Changing search bar focus is uncomfortable for users (you need to click on mod column itself to kill search focus, clicking on mod panel does nothing). So this code will be removed.